banking-app/test_docker.sh

```diff
@@ -0,0 +1,231 @@
+cd workspace/docker_certificates
+
+##############################################
+# Activate member 0
+##############################################
+echo "Getting list of members..."
+curl $server/gov/members \
+    --cacert service_cert.pem \
+    | jq
+
+curl "${server}/gov/ack/update_state_digest" \
+    -X POST \
+    --cacert service_cert.pem \
+    --key member0_privk.pem \
+    --cert member0_cert.pem \
+    --silent | jq > activation.json
+
+echo "Show digest"
+cat activation.json
+
+/opt/ccf/bin/scurl.sh "${server}/gov/ack" \
+    --cacert service_cert.pem \
+    --signing-key member0_privk.pem \
+    --signing-cert member0_cert.pem \
+    --header "Content-Type: application/json" \
+    --data-binary @activation.json
+
+echo "Getting list of members..."
+curl ${server}/gov/members \
+    --cacert service_cert.pem \
+    | jq
+
+##############################################
+# Generate Proposals
+##############################################
+create_test_network_proposal
+
+##############################################
+# Open Network
+##############################################
+echo "Open the network"
+network_proposal_out=$(/opt/ccf/bin/scurl.sh "${server}/gov/proposals" \
+    --cacert service_cert.pem \
+    --signing-key member0_privk.pem \
+    --signing-cert member0_cert.pem \
+    --data-binary @network_open_proposal.json \
+    -H "content-type: application/json")
+echo ${network_proposal_out} | jq
+network_proposal_out_id=$( jq -r  '.proposal_id' <<< "${network_proposal_out}" )
+
+echo "Network Proposal ID: $network_proposal_out_id"
+/opt/ccf/bin/scurl.sh "${server}/gov/proposals/$network_proposal_out_id/ballots" \
+    --cacert service_cert.pem \
+    --signing-key member0_privk.pem \
+    --signing-cert member0_cert.pem \
+    --data-binary @vote_accept.json \
+    -H "content-type: application/json" | jq
+
+##############################################
+# Propose users and application
+##############################################
+application_proposal_out=$(/opt/ccf/bin/scurl.sh "${server}/gov/proposals" \
+    --cacert service_cert.pem \
+    --signing-key member0_privk.pem \
+    --signing-cert member0_cert.pem \
+    --data-binary @../../dist/set_js_app.json \
+    -H "content-type: application/json")
+application_proposal_out_id=$( jq -r  '.proposal_id' <<< "${application_proposal_out}" )
+echo "Application Proposal ID: ${application_proposal_out_id}"
+/opt/ccf/bin/scurl.sh "${server}/gov/proposals/${application_proposal_out_id}/ballots" \
+    --cacert service_cert.pem \
+    --signing-key member0_privk.pem \
+    --signing-cert member0_cert.pem \
+    --data-binary @vote_accept.json \
+    -H "content-type: application/json" | jq
+
+##############################################
+# Test Network
+##############################################
+curl "${server}/node/network" --cacert service_cert.pem | jq
+
+# -------------------------- Preparation --------------------------
```

Comment:
Is it possible just to run/call test.sh rather than to copy the contents?

Reply:
I've just reworked the whole PR - I hope this is better - let me know your thoughts